src/sentry/replays/endpoints/organization_replay_selector_index.py

```diff
@@ -301,12 +301,16 @@ def process_raw_response(response: list[dict[str, Any]]) -> list[dict[str, Any]]
     """Process the response further into the expected output."""
 
     def make_selector_name(row) -> str:
-        selector = row["click_tag"]
-
-        if row["click_id"]:
-            selector = selector + f"#{row['click_id']}"
-        if row["click_class_filtered"]:
-            selector = selector + "." + ".".join(row["click_class_filtered"])
+        if row["click_component_name"]:
+            selector = row["click_component_name"]
+            if row["click_id"]:
+                selector = selector + f"#{row['click_id']}"
```

Comment:
Is there a reason we don't want to include the classes?  They are part of the aggregation key as well.

Reply:
we want to switch to use component name if that's available since the class is usually autogenerated in react and not descriptive

Reply:
We're still aggregating by class.  We're hiding the true name of the aggregation key not altering the aggregation.  We should remove class aggregation if its not valuable.

Reply:
Ah, can we generate a "display name" type selector on the frontend and keep the selector the same?

Reply:
I think we want to be able to search by the component name in the r&d selectors too, which would require changing the selectors.

Reply:
Searching is a different problem to aggregation.  We can search by component name regardless of this change.

If we're aggregating two rows:

| Component Name | Classes |
| -- | -- |
| SignUpForm | [blue, square] |
| SignUpForm | [red, circle] |

Should we return:

```javascript
["SignUpFrom", "SignUpForm"]
```

Or should we return:

```javascript
["SignUpFrom"]
```


Or should we return:

```javascript
["SignUpFrom.blue.square", "SignUpForm.red.circle"]
```

Reply:
We are keeping the selector mostly the same, except with an added component name attribute, and we will generate a "display name" type selector in the frontend